src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -48,6 +49,10 @@ export default function AppointmentsPage() {
   const [hasTypeChanged, setHasTypeChanged] = useState(false);
   let [pageTitle] = useState('VA online scheduling');
 
+  const featureAppointmentDetailsRedesign = useSelector(state =>
```

Comment:
Can we change the name `featureCCDirectScheduling`

Reply:
Also can we update the selector in src/applications/vaos/referral-appointments/components/ReferralAppLink.jsx to `selectFeatureCCDirectScheduling` .